pypesto/ensemble/ensemble.py

```diff
@@ -1157,3 +1184,33 @@ def get_percentile_label(percentile: Union[float, int, str]) -> str:
             percentile_str += '...'
         percentile = percentile_str
     return f'{PERCENTILE} {percentile}'
+
+
+def calculate_cutoff(result: Result, percentile: float = 0.95):
+    """
+    Calculate the cutoff of the ensemble.
+
+    Based on the number of parameters of the problem. Based on the
+    assumpiton that the difference of the NLLs of the true and optimal
+    parameter is chi^2 distributed with n_theta degress of freedom.
```

Comment:
- `nllh` elsewhere in this PR, `NLL` here
- `n_theta` could be changed e.g. "chi^2 distribution with degrees of freedom equal to the number of estimated parameters"
- typo "assumption"